app/api/routes/image_queries.py

```diff
@@ -210,15 +210,3 @@ async def post_image_query(  # noqa: PLR0913, PLR0915, PLR0912
         confidence_threshold=confidence_threshold,
         human_review=human_review,
     )
-
-
-@router.get("/{id}", response_model=ImageQuery)
-async def get_image_query(
```

Comment:
We don't need to do any of the IQE handling here anymore, so we can just let this fall back to the cloud.